From 1ea303d92a99011ad67552769c479134c3e66e67 Mon Sep 17 00:00:00 2001
From: "Jeroen van Meeuwen (Kolab Systems)" <vanmeeuwen@kolabsys.com>
Date: Sun, 25 Nov 2012 13:21:44 +0000
Subject: [PATCH 3/3] Import os before using it

---
 pykolab/setup/__init__.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pykolab/setup/__init__.py b/pykolab/setup/__init__.py
index a544e2a..2d9fafe 100644
--- a/pykolab/setup/__init__.py
+++ b/pykolab/setup/__init__.py
@@ -17,6 +17,7 @@
 # Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 #
 
+import os
 import sys
 
 import pykolab
-- 
1.7.11.7

